### PR TITLE
Stats: Rejection reasons page

### PR DESCRIPTION
**Pull request by mtuchowski**
Here we have a new table : the "table of reasons of rejections" : http://3m5fo4.axshare.com/#p=4__table_of_reasons_of_rejections

The star icon means that the reason is a systemic reason (= one of the options in the rejection reasons dropdowns).

We would like to : 

* be able to sort the columns by clicking on each columns title
* be able to see the file when clicking on each file name

### CODE REVIEW

**Comment by mtuchowski** [comment]
Adjust in end systems:

* https://github.com/egovernment/eregistrations-salvador/issues/1754
* https://github.com/egovernment/eregistrations-guatemala/issues/1056
* https://github.com/egovernment/eregistrations-tanzania/issues/1063

**Comment by mtuchowski** [comment]
There is a test port available for this ticket in ELS system, check https://github.com/egovernment/eregistrations-salvador/pull/1754 for details.

File: server/business-process-query/business-processes/filter.js [hunk 1]
```diff
@@ -54,14 +56,28 @@ module.exports = exports = function (data, query) {
 			if (bpData.submitterType !== query.submitterType) return;
 		}
 
-		// Filter by approved in given date range
+		// Select data filter subject
+		if (query.flowStatus) {
```
Comment (kamsi) [inline server/business-process-query/business-processes/filter.js]: This largely duplicates logic from 29. Let's include it there.

File: server/routes/inspector.js [hunk 2]
```diff
@@ -37,31 +37,31 @@ module.exports = exports = function (config) {
...
-				data = sortData(
-					filterBusinessProcesses(data.businessProcesses, query),
-					function (bpA, bpB) {
-						return bpA.createdDateTime.getTime() - bpB.createdDateTime.getTime();
-					}
-				);
+					data = sortData(
+						filterBusinessProcesses(data.businessProcesses, query),
+						function (bpA, bpB) {
+							return bpA.createdDateTime.getTime() - bpB.createdDateTime.getTime();
```
Comment (kamsi) [inline server/routes/inspector.js]: Let's remove `getTime()`

**Comment by mtuchowski** [comment]
I also updated the adjusts.

**Comment by kamsi** [comment]
@vianneyl this one has passed review, so it can be merged as soon as the tests are passed.

**Comment by mtuchowski** [comment]
@kamsi Ready to merge.

**Pull request merged**